test/parallel/test-performance-function.js

```diff
@@ -43,27 +43,6 @@ const {
   obs.disconnect();
 }
 
-{
-  class N {}
-  const n = performance.timerify(N);
-
-  const obs = new PerformanceObserver(common.mustCall((list) => {
-    const entries = list.getEntries();
-    const entry = entries[0];
-    assert.strictEqual(entry[0], 1);
-    assert.strictEqual(entry[1], 'abc');
-    assert(entry);
-    assert.strictEqual(entry.name, 'N');
-    assert.strictEqual(entry.entryType, 'function');
-    assert.strictEqual(typeof entry.duration, 'number');
-    assert.strictEqual(typeof entry.startTime, 'number');
-    obs.disconnect();
-  }));
-  obs.observe({ entryTypes: ['function'] });
-
-  new n(1, 'abc');
-}
-
```

Comment:
Is there a reason for disallowing classes in timerify()? Reading the review comments in https://github.com/nodejs/node/pull/40625, I get the feeling that we still do want to keep supporting classes here. Could you please add this test back?